lib/swagger/formatter.ex

```diff
@@ -57,71 +38,62 @@ defmodule Xcribe.Swagger.Formatter do
     }
   end
 
-  defp format_body_params(body, request) when is_map(body) do
-    body
-    |> Map.to_list()
-    |> format_params(request)
+  defp include_params(list, params, request, inn) do
```

Comment:
```suggestion
  defp include_params(acc, params, request, inn) do
```

:nail_care:

Reply:
At next line there is a `Enum.reduce` with the same var name `acc`. =/